### ISSUE TITLE
proposer un nouveau design pour la page "stop_rappel"

### ISSUE THREAD

**Issue by axel584** [axel584/ikurso#289]
http://ikurso.esperanto-france.org/stop_rappel.php?id=7780

Je pense que ça peut être très sympa à faire, peut être ajouter un lien vers la page d'accueil et/ou la page de données personnelles ?

**Comment by emmrichard** [comment]
Ce serait bien sur la page Données personnelles, dont on peut rappeler le lien dans le mail qu'on envoie.
@axel584 : est-ce qu'il suffit de mettre un lien vers la page stop_rappel ? Est-ce que cette page pourrait juste afficher un toast ?

**Comment by axel584** [comment]
Oui, avec un toast ça me semble bien, actuellement, on a un code "20" qui affiche "Les changements ont bien été enregistrés"... Est ce qu'on affiche la même chose ?
Je pense qu'il faudrait ajouter un peu de sécurité, par exemple en envoyant le code d'activation, pour être sûr que la personne qui clique sur le lien est bien celle qui l'a reçu et pas quelqu'un qui s'est amusé à modifier l'url en essayant les différentes valeurs de l'id.

Du coup, je peux m'en occuper tout seul.
